### ISSUE TITLE
Internet access form containers problem

### ISSUE THREAD

**Issue by cdupont** [Waziup/WaziGate#34]
I got this message:
![Screenshot from 2019-11-15 12-47-12](https://user-images.githubusercontent.com/484033/68941427-254d2600-07a6-11ea-9da8-51d6f31644dc.png)

But the internet seems accessible:
![Screenshot from 2019-11-15 12-47-18](https://user-images.githubusercontent.com/484033/68941429-267e5300-07a6-11ea-8635-3841569ead7b.png)

After this, I cannot connect anymore with the tunnel.

**Comment by mojtaba-esk** [comment]
Yes we design it this way and this is not a bug. When there is no internet, how we can check the credentials?

**Comment by cdupont** [comment]
This issue came back. It seems that the host can access internet, but not the containers.
https://stackoverflow.com/questions/20430371/my-docker-container-has-no-internet

Probably the solution is to set the DNS on the host file resolv.conf:
```
nameserver 8.8.8.8
nameserver 8.8.4.4
```
